docs/rules/require-hook.md

```diff
@@ -148,3 +148,61 @@ afterEach(() => {
   clearCityDatabase();
 });
 ```
+
+## Options
+
+If there are methods that you want to call outside of hooks and tests, you can
+mark them as allowed using the `allowedFunctionCalls` option.
+
+```json
+{
+  "jest/require-hook": [
+    "error",
+    {
+      "allowedFunctionCalls": ["enableAutoDestroy"]
+    }
+  ]
+}
+```
+
+Examples of **incorrect** code for the
+`{ "allowedFunctionCalls": ["enableAutoDestroy"] }` option:
```

Comment:
nit: I think it might make more sense to say:

```suggestion
Examples of **incorrect** code when using the `allowedFunctionCalls` option:
```

since we've got the config straight after in a comment